demo/full/styles/style.css

```diff
@@ -752,6 +772,35 @@ body {
   background-position: calc(100% - 8px) 50%;
 }
 
+.generated-url {
+  color: #3a3a3a;
+  font-size: 0.8em;
+  background-color: rgba(255, 255, 255, 0.8);
+  border-radius: 3px;
+  padding: 10px;
+  display: none;
+  -webkit-transition: opacity 2s;
+  transition: opacity 2s;
```

Comment:
There is no visible transition in the demo. So either:
  - we fix that
  - we remove that rule